### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by casserole27**
[Wave Report](https://wave.webaim.org/report#/https://www.clewisdev.com/scrimba-restaurant-app/)

Changed discount info to a heading.
Added ARIA labels to the plus and minus buttons so they are more clear for screen readers.

### CODE REVIEW

**Comment by Rapidisimo** [comment]
I'm seeing the following issue on my end:
image

File: index.js [hunk 1]
```diff
@@ -47,8 +47,16 @@ function renderMenu() {
                 <button class="add-btn" data-item="${item.id}">+</button>
```
Comment (Rapidisimo) [inline index.js:47]: This seems like a duplicate.

File: index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
      
       </section>
 
-      <p class="discount-info">Buy two or more items, get 15% off your order!</p>
+      <h3 class="discount-info">Buy two or more items, get 15% off your order!</h3>
```
Comment (Rapidisimo) [inline index.html:33]: This is also a duplicate.

**Review by Rapidisimo — CHANGES_REQUESTED** [review]
Remove duplicate buttons

**Comment by casserole27** [comment]
Removed duplicate buttons and merged with main.

**Pull request merged**